tests/docker/config/construct-query.sparql

```diff
@@ -46,90 +66,126 @@ CONSTRUCT {
+                             ?age                   :value       ?subjectAgeValue    ;
+                                                    :minValue    ?subjectAgeMinValue ;
+                                                    :maxValue    ?subjectAgeMaxValue ;
+                                                    :unit        ?subjectAgeUnit     ;
+                                                    :period      ?subjectAgePeriod   ;
+                                                    :label       ?subjectAgeLabel    .
+  ?id  :subjectWeight        ?subjectWeightBN       .
+                             ?subjectWeightBN       :value       ?subjectWeightValue     ;
+                                                    :unit        ?subjectWeightUnit      ;
+                                                    :minValue    ?subjectWeightMinValue  ;
+                                                    :maxValue    ?subjectWeightMaxValue  ;
+                                                    :label       ?subjectWeightLabel     .
+
+  # Properties specific to types
+
+  ## Bouton density
+  ?id  :boutonDensity      ?boutonDensityBN            .
+                           ?boutonDensityBN            :value       ?boutonDensityValue  ;
+                                                       :unit        ?boutonDensityUnit   ;
+                                                       :label       ?boutonDensityLabel  .
+
+  ## Circuit
+  ?id  :circuitType          ?circuitType              ;
+       :circuitBase          ?circuitBaseUrlStr        ;
+       :circuitConfigPath    ?circuitConfigPathUrlStr  ;
+       :circuitBrainRegion   ?circuitBrainRegionBN     .
+                             ?circuitBrainRegionBN     :identifier  ?circuitBrainRegionId    ;
+                                                       :label       ?circuitBrainRegionLabel .
+
+  ## Layer thickness
+  ?id  :layerThickness     ?thicknessBN           .
+                           ?thicknessBN           :value       ?thicknessValue   ;
+                                                  :unit        ?thicknessUnit    ;
+                                                  :nValue      ?thicknessNValue  ;
+                                                  :label       ?thicknessLabel   .
+
+  ## Neuron density
+  ?id  :neuronDensity      ?neuronDensityBN       .
+                           ?neuronDensityBN       :value       ?neuronDensityValue   ;
+                                                  :unit        ?neuronDensityUnit    ;
+                                                  :nValue      ?neuronDensityNValue  ;
+                                                  :label       ?neuronDensityLabel   .
+
+  ## Simulation campaigns
+  ?id  :config               ?campaignConfigId        .
+                             ?campaignConfigId        :identifier      ?campaignConfigId   ;
+                                                      :label           ?campaignConfigName .
```

Comment:
Should we consider using `:name` here? I know it will not be consistent with the rest but we should refrain from switching the names of field when adding new properties at least. IMO without any doc users expect (and are kind of expected) to find search fields using the original resource structure